src/start_server.ts

```diff
@@ -100,11 +100,17 @@ export function getApp(options: ServerOptions) {
 
   app.use('/components/', polyserve);
 
+  const isFileRequest = (filePath: String) => {
+    return ['html', 'js', 'json', 'css', 'png', 'jpg', 'jpeg', 'gif'].filter((ext) => {
```

Comment:
Yea I agree it's not perfect either, but I like it better than the explicit list. Lets keep it simple at first and match all url paths where an extension (period & 1+ other characters) exists after the last slash.